docs/topics/clusterdefinitions.md

```diff
@@ -51,7 +51,7 @@ $ aks-engine get-versions
 | enableAggregatedAPIs            | no       | Enable [Kubernetes Aggregated APIs](https://kubernetes.io/docs/concepts/api-extension/apiserver-aggregation/).This is required by [Service Catalog](https://github.com/kubernetes-incubator/service-catalog/blob/master/README.md). (boolean - default is true for k8s versions greater or equal to 1.9.0, false otherwise)                                                                                                                                              |
 | enableDataEncryptionAtRest      | no       | Enable [kubernetes data encryption at rest](https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/).This is currently an alpha feature. (boolean - default == false)                                                                                                                                                                                                                               |
 | enableEncryptionWithExternalKms | no       | Enable [kubernetes data encryption at rest with external KMS](https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/).This is currently an alpha feature. (boolean - default == false)                                                                                                                                                                                                             |
-| enablePodSecurityPolicy         | no       | Enable [kubernetes pod security policy](https://kubernetes.io/docs/concepts/policy/pod-security-policy/).This is currently a beta feature. (boolean - default == false)                                                                                                                                                                                                                                       |
+| enablePodSecurityPolicy         | no       | Enable [kubernetes pod security policy](https://kubernetes.io/docs/concepts/policy/pod-security-policy/).This is currently a beta feature. (boolean - default == false for Kubernetes 1.14 and earlier, true for Kubernetes v1.15+)                                                                                                                                                                                                 |
```

Comment:
this implies that the _default_  is true for 1.15+ when in fact (at least with how the code is written right now) this value gets set to true statically for all 1.15+ clusters, no matter what the apimodel `enablePodSecurityPolicy` value is. We should either make that clear or change the code to honor custom values.